tf2_eigen/package.xml

```diff
@@ -8,13 +8,13 @@
   <license>BSD</license>
 
   <buildtool_depend>catkin</buildtool_depend>
+
+  <build_depend>cmake_modules</build_depend>
```

Comment:
I think you're right. All the EIGEN variables will be evaluated so it won't need them downstream.